cleanlab/multiannotator.py

```diff
@@ -0,0 +1,829 @@
+import numpy as np
+import pandas as pd
+from typing import List, Union, Tuple
+from cleanlab.rank import get_label_quality_scores
+from cleanlab.internal.util import get_num_classes
+from cleanlab.internal.label_quality_utils import _get_label_quality_scores_with_NA
+from cleanlab.dataset import _get_worst_class
+import warnings
+
+
+def convert_long_to_wide_dataset(
+    labels_multiannotator_long: pd.DataFrame,
+) -> pd.DataFrame:
+    """Converts a wide format dataset to long format which is suitable for passing into
+    :py:func:`get_label_quality_multiannotator <cleanlab.multiannotator.get_label_quality_multiannotator>`.
+    Dataframe must contain three columns named:
+    1. ``task`` representing each example labeled by the annotators
+    2. ``annotator`` representing each annotator
+    3. ``label`` representing the label given by an annotator for the corresponding task
+
+    Parameters
+    ----------
+    labels_multiannotator_long : pd.DataFrame
+        pandas DataFrame in long format with three columns named ``task``, ``annotator`` and ``label``
+    """
+    labels_multiannotator = labels_multiannotator_long.pivot(
+        index="task", columns="annotator", values="label"
+    )
+    labels_multiannotator.index.name = None
+    labels_multiannotator.columns.name = None
+    return labels_multiannotator
+
+
+def get_consensus_label(
+    labels_multiannotator: pd.DataFrame or np.ndarray,
+    pred_probs: np.ndarray = None,
+    consensus_method: str = "majority",
+) -> np.ndarray:
+    """Returns consensus labels aggregated from multiple annotators, determined by method specified.
```

Comment:
could rephrase as:

Returns a single consensus label for each example, aggregated from the labels given by multiple annotators.

Reply:
The discussion of which method is used for the aggregation should IMO be reserved for the proper parameters below. We don't want non-advanced users to already be thinking about methods/choices they have to make. We prefer the package seems easy-to-use and trusted to already be using good default settings.